### ISSUE TITLE
Keyboard Navigation

### ISSUE THREAD

**Issue by allanlasser** [MuckRock/documentcloud-frontend#779]
A common pro-user request is for keyboard shortcuts to make navigating the UI faster. Keyboard navigation will also improve accessibility. 

We already use keyboard navigations in Dropdowns, although our approach can be improved. [@eyeseast on PR #776](https://github.com/MuckRock/documentcloud-frontend/pull/776#discussion_r1797397667): 

> At some point we should move these events out of lifecycle and into <svelte:document> event handlers, but that doesn't have to happen in this PR.

Successfully adding keyboard navigation will include:

- designing a mapping between app functions and keys
- designating a "control" key, if needed

Since keyboard handlers will be attached to the document, they should be designed for global scope.
